bessctl/commands.py

```diff
@@ -530,6 +534,10 @@ def _do_reset(cli):
 
 @cmd('daemon reset', 'Remove all ports and modules in the pipeline')
 def daemon_reset(cli):
+    if not is_bess_connected(cli):
```

Comment:
Yes, I think so. I couldn't think of a way to implement that without intricate the code. Can think about it in a next PR.